os/access.c

```diff
@@ -1073,7 +1073,7 @@ ComputeLocalClient(ClientPtr client)
          */
         char *tok = strtok(cmd, ":");
 
-#if !defined(WIN32)
+#if !defined(WIN32) || defined(__CYGWIN__)
```

Comment:
maybe we should just implement basename() on win32 instead and git rid of that #ifdef entirely ?